spec/requests/webhooks_spec.rb

```diff
@@ -24,7 +24,51 @@
         line_items: line_items
       )
     end
-    let(:seller) { create :seller }
+    let!(:seller_1) do
+      Seller.create(
+          seller_id: 'shunfa-bakery',
+          cuisine_name: 'Chinese',
+          name: 'Shunfa Bakery',
+          story: 'I am but a small, small boy',
+          owner_name: 'Ben Jerry',
+          owner_image_url: 'https://www.aws.com/98nuw9e8unf9awnuefaiwenfoaijfosdf',
+          accept_donations: true,
+          sell_gift_cards: true,
+          business_type: 'small-biz',
+          num_employees: 5,
+          founded_year: 1850,
+          website_url: 'https://www.youtube.com/watch?v=jIIuzB11dsA',
+          menu_url: 'https://www.youtube.com/watch?v=jIIuzB11dsA',
+          square_location_id: '1234-abcd'
+      )
+    end
+    let!(:seller_2) do
+      Seller.create(
+          seller_id: '87-lan-zhou-handpooled-noods',
+          cuisine_name: 'Noodle Soup',
+          name: '87 Lan Zhou Handpooled Noods',
+          story: 'Been pullin noods since I was 2',
+          owner_name: 'Tom Hanks',
+          owner_image_url: 'https://www.aws.com/oawjeoiajwef9wuef09wuef09waeuf',
+          accept_donations: false,
+          sell_gift_cards: true,
+          business_type: 'medium-biz',
+          num_employees: 10,
+          founded_year: 1950,
+          website_url: 'https://www.youtube.com/watch?v=C_oACPWGvM4',
+          menu_url: 'https://www.youtube.com/watch?v=C_oACPWGvM4',
+          square_location_id: '4567-efgh'
+      )
+    end
+    let!(:seller_pool) do
+      Seller.create(
```

Comment:
Could you add this into a separate test seeds.rb?

Reply:
Is that something we can hook up to actually seed all tests or just a file people can copy out of manually to seed their own tests?

Reply:
It'll seed all tests

Reply:
We'll need it here because we should always expect this "special seller" to exist on all environments